documentation/api/folders/listdocuments.yaml

```
@@ -0,0 +1,53 @@
+---
+get:
+  summary: List Documents in Folder
+
+  tags:
+    - Folders
+
+  description: |
+    > Example of a Folder Listing
+
+    ```json
+    {
+      "total_matching": 2,
+      "documents": [
+        <Document JSON (see doc endpoints)>,
+        <Document JSON (see doc endpoints)>
+      ]
+    }
+    ```
+
+    This endpoint is used to get the details of the documents in a given folder.
+
+    Other than the JSON fields shown above, which wrap the output data in a list
+    and show a little metadata, the document JSON for each item in `"documents"`
+    is structured the same as it would be for endpoints that display a given
+    document. _(See the [Document Definition](#document-3) documentation for
+    details)._
+
+  parameters:
+    - name: folder_id
+      in: url
+      type: integer
```

the ID in the `Folder`-type is a `string`

```suggestion
      type: string
```